src/components/app-body.js

```diff
@@ -5,6 +5,11 @@ import React, {Component} from 'react';
 import LoginJunk from './login-junk.js';
 
 
+
+const FancyInput = (props) => {
+		return <label> {props.label} <input type="text" /></label>;
```

Comment:
also, `FancyInput` needs one more prop: `name` which gives the form input a name that we can use to extract the form value i.e. `<input name={props.name} type="text" />`